pkg/cmd/openshift/openshift.go

```diff
@@ -97,18 +97,15 @@ func CommandFor(basename string) *cobra.Command {
 func NewCommandOpenShift(name string) *cobra.Command {
 	in, out, errout := os.Stdin, os.Stdout, os.Stderr
 
-	product := "Origin"
-	switch name {
-	case "openshift":
-		product = "OpenShift"
-	case "atomic-enterprise":
-		product = "Atomic"
+	distPrefix := cmdutil.GetProductName(name)
+	if distPrefix == cmdutil.ProductAtomicEnterprise {
+		distPrefix = "Atomic"
```

Comment:
why are we doing this transformation here? I want this logic in one place... if we have distinct distribution, product, and platform names (which seems a little crazy), there should be three helpers, `GetDistributionName`, etc